.storybook/config.js

```diff
@@ -16,7 +16,7 @@ addDecorator(withA11y);
 import '../src/components/styles.scss';
 
 // automatically import all files ending in *.stories.js
-configure(require.context('../src/components', true, /\.stories\.js$/), module);
+configure(require.context('western-up-react', true, /\.stories\.js$/), module);
```

Comment:
@ModulesUnraveled I had to change this from `western-up-react ` to `../western-up-react` for storybook to discover the western up stories. Not sure if anyone else will hit this roadblock or if I set up the site incorrectly.

Reply:
Assuming we're not using workspaces, we should just be able to use `western-up-react`

Reply:
I've been using workspaces and it works as is. I know what the problem was for you Jim.